src/EIP7702Proxy.sol

```diff
@@ -107,13 +109,11 @@ contract EIP7702Proxy is Proxy {
         }
     }
 
+    /// @review Do we need to override this now that we are setting the implementation at initialization?
```

Comment:
my reasoning for leaving the override was that just because a contract isn't initialized yet, it could still be called, in which case we need to delegate those calls to the initial implementation.  maybe this can be removed / wouldn't make sense anymore if we prevent all calls to an uninitialized contract, but I don't really see the benefit of doing that since it just prevents reads but doesn't really create any additional security